### PR TITLE
Update previous-versions.md

### PR DESCRIPTION
**Pull request by svekars**
Updating previous-version.md to include 1.12.0 release.

### CODE REVIEW

**Review by atalman — COMMENTED** [review]
Looks good for 1.12.0
Should we add 1.12.1 here as well ?

File: _get_started/previous-versions.md [hunk 1]
```diff
@@ -17,6 +17,53 @@ your convenience.
...
+# conda
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 -c pytorch
+```
+
+#####  Linux and Windows
+
+```
+# CUDA 10.2
+conda install pytorch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0 cudatoolkit=10.2 -c pytorch
+# CUDA 11.3
```
Comment (atalman) [inline _get_started/previous-versions.md:36]: We are missing 11.6 CUDA for 1.12.0

File: _get_started/previous-versions.md [hunk 2]
```diff
@@ -17,6 +17,53 @@ your convenience.
...
+##### Linux and Windows
+
+```
+# ROCM 4.2 (Linux only)
+pip install torch==1.12.0+rocm4.2 torchvision==0.13.0+rocm4.2 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# ROCM 4.1 (Linux only)
+pip install torch==1.12.0+rocm4.1 torchvision==0.13.0+rocm4.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# ROCM 4.0.1 (Linux only)
+pip install torch==1.12.0+rocm4.0.1 torchvision==0.13.0+rocm4.0.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# CUDA 11.1
```
Comment (agunapal) [inline _get_started/previous-versions.md:59]: Is this supported with CUDA 11.1 or 11.3?

**Comment by svekars** [comment]
@atalman I thought 1.12.1 is still current and described here: https://pytorch.org/get-started/locally/. Should it be added after the next release?

**Comment by atalman** [comment]
> @atalman I thought 1.12.1 is still current and described here: https://pytorch.org/get-started/locally/. Should it be added after the next release?

Yes, correct, my bad. Please add some description to PR, so its clear why we are doing this.

File: _get_started/previous-versions.md (hunk 2 above)
Comment (atalman) [reply _get_started/previous-versions.md:59]: Yes true cu111 is dropped since 1.11.0 I believe

File: _get_started/previous-versions.md (hunk 1 above)
Comment (svekars) [reply _get_started/previous-versions.md:36]: added below

File: _get_started/previous-versions.md [hunk 3]
```diff
@@ -17,6 +17,53 @@ your convenience.
...
+
+##### Linux and Windows
+
+```
+# ROCM 5.1.1 (Linux only)
+pip install torch==1.12.0+rocm5.1.1 torchvision==0.13.0+rocm5.1.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# CUDA 11.6
+pip install torch==1.12.0+cu102 torchvision==0.13.0+cu116 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# CUDA 11.3
+pip install torch==1.12.0+cu111 torchvision==0.13.0+cu113 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (agunapal) [inline _get_started/previous-versions.md:60]: Should be cu113

File: _get_started/previous-versions.md [hunk 4]
```diff
@@ -17,6 +17,53 @@ your convenience.
...
+pip install torch==1.12.0 torchvision==0.13.0 torchaudio==0.12.0
+```
+
+##### Linux and Windows
+
+```
+# ROCM 5.1.1 (Linux only)
+pip install torch==1.12.0+rocm5.1.1 torchvision==0.13.0+rocm5.1.1 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
+# CUDA 11.6
+pip install torch==1.12.0+cu102 torchvision==0.13.0+cu116 torchaudio==0.12.0 -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (agunapal) [inline _get_started/previous-versions.md:58]: should be cu116

**Review by agunapal — APPROVED** [review]
looks good to me

**Comment by svekars** [comment]
I don't this PR affected that part of the page. We only added instructions for 1.12 but did not touch the 1.10.1 part.

**Pull request merged**